lib/server.go

```diff
@@ -490,6 +483,36 @@ func NewServer(
 	_mempool := NewDeSoMempool(_chain, _rateLimitFeerateNanosPerKB,
 		_minFeeRateNanosPerKB, _blockCypherAPIKey, _runReadOnlyUtxoViewUpdater, _dataDir,
 		_mempoolDumpDir, false)
+	_posMempool := NewPosMempool()
+
+	// Initialize the PoS mempool. We need to initialize a best-effort UtxoView based on the current
+	// known state of the chain. This will all be overwritten as we process blocks later on.
+	currentUtxoView, err := _chain.GetUncommittedTipView()
+	if err != nil {
+		return nil, errors.Wrapf(err, "NewServer: Problem initializing latest UtxoView"), true
+	}
+	currentGlobalParamsEntry := currentUtxoView.GetCurrentGlobalParamsEntry()
+	latestBlockHash := _chain.blockTip().Hash
+	latestBlock := _chain.GetBlock(latestBlockHash)
+	if latestBlock == nil {
+		return nil, errors.New("NewServer: Problem getting latest block from chain"), true
+	}
+	err = _posMempool.Init(
+		_params,
+		currentGlobalParamsEntry,
+		currentUtxoView,
+		uint64(_chain.blockTip().Height),
+		_mempoolDumpDir,
+		false,
+		1024*1024*1024*3, // Max mempool Size = 3GB; TODO make this a param
+		60*1000,          // Mempool dumper frequency = 60 seconds; TODO make this a param
```

Comment:
Maybe move these to `constants.go` for now?

Reply:
Eh, this will all be dead code in a few PRs. You think it's worth creating consts elsewhere in the code for them?

Reply:
nah screw it.